README.md

```diff
@@ -1,18 +1,680 @@
+
+- Incorrect: "You can provide an order ID or customer id as a Workflow Id."
+
+In code (and when quoting or referring to code in text), follow the conventions of each language.
+
+#### En dashes in ranges
+
+Using an en dash (`&ndash;` or the character `–`) in a range of numbers is acceptable.
+Even better is to use words such as _from_, _to_, and _through_.
+
+#### Be consistent.
+
+If you use an en dash in one range, use en dashes in all ranges.
+Do not mix words and en dashes (or hyphens, for that matter).
+
+- Correct: "5 to 10 GB"
+- Correct: "5–10 GB"
+- Correct: "5-10 GB"
+- Incorrect: "from 5-10 GB"
+
+#### Infinitive verb forms in headings
+
+Titles and headings should use infinitive verb forms whenever possible. People tend to search by using infinitive verb forms, so using them helps SEO.
+
+- Correct: "How to install Temporal"
+- Incorrect: "Installing Temporal"
+
+#### Infinitive verb forms in labels
+
+- Correct: "Install Temporal"
+- Incorrect: "Installing Temporal"
+
+#### Sentence casing in headings
+
+Use sentence casing for titles and headings.
+Sentence casing means that only the first letter of the first word and proper nouns are capitalized.
+
+- Correct: "How to get started with Temporal"
+- Incorrect: "How To Get Started With Temporal"
+
+### What is the philosophy around versioning the documentation?
+
+The Temporal Platform includes many different components and core dependencies. Each of these components is often independently versioned and documents their stability and support for their own dependencies. For example, the [Temporal Go SDK reference](https://pkg.go.dev/go.temporal.io/sdk?tab=versions) provides that view.
+
+The goal of this information set, in regards to versioning, is to remain “current”. That is, this information should serve the needs of the Platform’s user base as best it can based on what is/has recently happened across all the components of the Temporal Platform.
+We should explicitly call out support, stability, and dependency information whenever possible inline with the rest of the information. This contrasts with publishing versions of documentation sets to match the versioned components of the platform.
+
+We use the `ssdi` metadata tag in `docs-src` nodes to explicitly call out support, stability, and dependency information when needed.
+
+#### How to explicitly identify support, stability, and dependency info
+
+Use the `ssdi` metadata tag.
+
+Add this metadata tag to the information node’s metadata tag in the `/docs-src` directory.
+
+Example:
+
+```markdown
+---
+id: what-is-x
+// ...
+ssdi:
+  - Introduced in [Temporal Server version x.xx.x](/path/to/release/notes/)
+  - Available in [Temporal CLI version x.xx.x](/path/to/release/notes/)
+  - Released in Temporal Cloud on <date>
+---
+```
+
+### How to auto format files
+
+Run `yarn format` .
+
+If you do not run this command locally and put up a Pull Request, a Github action runs the formatting.
```

Comment:
```suggestion
If you don't run this command locally before posting a Pull Request, a Github action runs the formatting.
```

Reply:
Need more context for the GitHub action, especially why we'd rather format the files.